src/commands/switch.rs

```diff
@@ -0,0 +1,187 @@
+use super::super::errors;
+use super::*;
+use crate::core::Target;
+use crate::tasks::*;
+use crate::{core, git};
+use clap::{App, Arg};
+
+pub struct SwitchCommand {}
+
+impl Command for SwitchCommand {
+    fn name(&self) -> String {
+        String::from("switch")
+    }
+
+    fn app<'a>(&self) -> App<'a> {
+        App::new(self.name().as_str())
+            .version("1.0")
+            .alias("s")
+            .about("switches to the specified branch within the current repository")
+            .after_help(
+                "This command switches to the specified branch within the current repository.",
+            )
+            .arg(
+                Arg::new("branch")
+                    .about("The name of the branch to switch to.")
+                    .index(1),
+            )
+            .arg(
+                Arg::new("create")
+                    .short('c')
```

Comment:
Do you reckon that it's worth adding a `.long("create")` entry here as well (so that you can `--create` too)?

Also probably need to add this as a suggestion to the `complete()` list.

Reply:
Yes agreed! Both of these suggestions are also addressed in the latest PR 👍